environs/config/config.go

```diff
@@ -1522,6 +1542,44 @@ func (c *Config) validateMode() error {
 	return nil
 }
 
+// SSHAllow returns a slice of CIDRs from which machines in
+// this model will accept connections to the SSH service
+func (c *Config) SSHAllow() []string {
+	allowList, ok := c.defined[SSHAllowKey].(string)
+	if !ok {
+		return []string{"0.0.0.0/0"}
+	}
+	if allowList == "" {
```

Comment:
Did we want "" or a const like "none". I guess "" might be clear enough.

Reply:
When retrieving these config items we split on commas and return a of cidrs, so having a const like "non" here would probably be confusing